tools/test/examples/examples.json

```diff
@@ -4,465 +4,1261 @@
       "name": "mbed-os-example-blinky",
       "github": "https://github.com/ARMmbed/mbed-os-example-blinky",
       "mbed": [
-          "https://os.mbed.com/teams/mbed-os-examples/code/mbed-os-example-blinky"
+        "https://os.mbed.com/teams/mbed-os-examples/code/mbed-os-example-blinky"
```

Comment:
Who can edit this file? With this update, it would make sense to state who can change and how - adding/removing device from PR/nightly tests for instance. I expect users will find this file and edit it. We need a reference to state this is the way it should be done - add section to Readme.